.github/workflows/tests.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           pip install -e .
           pip install pip-licenses
           if pip-licenses | grep -v 'Artistic License' | grep -v LGPL | grep GNU; then echo 'Please avoid introducing *GPL dependencies'; false; fi
-          pip install coverage coveralls
+          pip install coverage
       - name: Run tests
         run: |
           coverage run --omit g2p/app.py --omit 'g2p/tests/*' --parallel-mode \
```

Comment:
Perhaps we want to not omit g2p/tests here after all, so that github won't complain about changes to the test suite supposedly not being covered.

Reply:
Absolutely agree, I consider it a critical CI bug that test omission stopped working.

Reply:
How is it your fault? I removed coveralls because I thought we didn't need it. Do you have any idea how to get the omit to work without installing coveralls?

Reply:
(Not that installing coveralls is onerous, I don't mind just leaving it there, although I really wish I understood why its presence makes a difference.)